netkan/setup.py

```diff
@@ -10,6 +10,7 @@
     author_email='techman83@gmail.com',
     packages=find_packages(),
     install_requires=[
+        'wheel',
         'boto3',
```

Comment:
Why are we adding 'wheel' as an install dependency here?

Reply:
Oh right. Pip tries to build wheels when installing all the dependencies.
`wheel` is already preinstalled in the Python3 Docker Container, but not in a venv for example, and pip will throw errors (or warnings, don't know anymore) if `wheel` isn't installed.
This affects the containers not at all (pip says dependency already satisfied), but helps venv users (I have a venv setup too, because my IDE likes having the imported packages installed for code inspection).

Reply:
I still use a venv for the project. Though it makes no difference to the container, adding a build dependency as something the project requires seems a bit wrong.

> install_requires is a setuptools setup.py keyword that should be used to specify what a project minimally needs to run correctly. When the project is installed by pip, this is the specification that is used to install its dependencies.

Maybe in the dev dependencies is more appropriate?

Reply:
I remember I added it because I thought it pip threw errors due to this and didn't want to install the other packages.
But checking back I'm not sure why it threw errors, it doesn't throw them anymore when I delete the venv folder and install again :man_shrugging:

I'm going to remove it again, or would you want to keep it in the dev deps?

Reply:
Yeah, I'd just drop it. It's neither a package dependency or a development one.